viite-UI/src/map/CalibrationPointMarker.js

```diff
@@ -2,16 +2,12 @@
   root.CalibrationPoint = function(data) {
     var cachedMarker = null;
     var cachedDirectionArrow = null;
-    var defaultMarkerGraphics = {
-      externalGraphic: "images/calibration-point.svg"
-    };
     var firstCustomCalibrationPointValue=4;
     var createCalibrationPointMarker = function () {
+      const marker = new ol.Feature({
```

Comment:
Is it OK to use ES6 features? Does this work with IE11?

Reply:
Viite does not support IE11. It is ok to use es6 and newer features